addons/account_bank_statement_import/demo/partner_bank.xml

```diff
@@ -3,7 +3,7 @@
     <data>
 
         <record id="ofx_partner_bank_1" model="res.partner.bank">
-            <field name="acc_number">00987654321</field>
+            <field name="acc_number">BE68539007547034</field>
             <field name="partner_id" ref="base.res_partner_2"></field>
             <field name="acc_type">bank</field>
```

Comment:
`acc_type` is a no stored field, it can be removed here.

cc https://github.com/odoo/odoo/pull/24172

Reply:
Thanks for the feedback @fmdl 
I guess both of our PR's can be reviewed and merged in separated to handle both cases!